export/export.go

```diff
@@ -0,0 +1,24 @@
+package export
+
+import (
+	"fmt"
+
+	"github.com/bitrise-io/go-utils/v2/env"
+)
+
+type Exporter struct {
```

Comment:
As far as I see, this won't make the env available for the next steps

Reply:
We could use the [export](https://github.com/bitrise-io/go-steputils/tree/master/export) package or the [stepenv](https://github.com/bitrise-io/go-steputils/tree/master/stepenv) package (if we want to make the new env available in the current environment too)